openupgrade_scripts/scripts/hr_fleet/16.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,35 @@
+from openupgradelib import openupgrade
+
+
+def create_drive_employee(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE fleet_vehicle_assignation_log
+        ADD IF NOT EXISTS driver_employee_id INT
+        """,
+    )
+
+    openupgrade.logged_query(
+        env.cr,
+        """
+        SELECT id, driver_employee_id
+        FROM fleet_vehicle
+        """,
+    )
+
+    for vehicle in env.cr.fetchall():
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE fleet_vehicle_assignation_log
+            SET driver_employee_id = %s
+            WHERE vehicle_id = %s
+            """,
+            (vehicle[1], vehicle[0]),
+        )
```

Comment:
@nguyenvietlam0640 như này là được mà nhỉ?
```suggestion
        """
        UPDATE fleet_vehicle_assignation_log log
            SET driver_employee_id = emp.id
        FROM hr_employee emp
        WHERE log.driver_id IS NOT NULL and log.driver_id = emp.address_home_id
        """,
    )
```

Reply:
vâng anh, con này là đời đầu em làm :laughing: